inductiva/simulation/simulator.py

```diff
@@ -39,29 +57,25 @@ def run(
             **kwargs: Additional keyword arguments to be passed to the
                 simulation API method.
         """
-        input_dir = files.resolve_path(input_dir)
-        if not input_dir.is_dir():
-            raise ValueError(
-                f"The provided path (\"{input_dir}\") is not a directory.")
-
-        if output_dir is None:
-            output_dir = input_dir.with_name(f"{input_dir.name}-output")
-            output_dir = files.get_timestamped_path(output_dir)
+        input_dir, output_dir = self._setup_dirs(input_dir, output_dir)
+        if pipeline is None:
+            params = [kwargs]
         else:
-            output_dir = files.resolve_path(output_dir)
+            params = [command.get_args() for command in pipeline]
```

Comment:
I think this way of defining `params` can be problematic...

First, the web API will have to be aware whether to extract commands or parameters from the `params` it receives.

Also, there doesn't seem to be a way to run a simulation by setting both a pipeline of commands and extra parameters.

I think this would be much cleaner if `api.run_simulation` accepted an extra parameter `pipeline` **or** if we passed the pipeline as a parameter, with `params["pipeline"] = pipeline`.

Let me know what you think. cc @luispcunha @IvanPombo